contrib/babelfishpg_tsql/src/pltsql_bulkcopy.h

```diff
@@ -50,7 +50,7 @@ typedef struct BulkCopyStateData
 	int			seq_index;		/* index for an identity column */
 	Oid			seqid;			/* oid of the sequence for an identity column */
 	int			rv_index;		/* index for a rowversion datatype column */
-
+	List	   *rteperminfos;	/* single element list of RTEPermissionInfo */
```

Comment:
Do we have a test for this? If not, please add basic happy path and error path test

Reply:
The check constraint is the happy path test case